itest/assets_test.go

```diff
@@ -586,6 +606,65 @@ func getAssetChannelBalance(t *testing.T, node *HarnessNode, assetID []byte,
 		balance.RemoteBalance.Sat
 }
 
+func fetchChannel(t *testing.T, node *HarnessNode,
+	chanPoint *lnrpc.ChannelPoint) *lnrpc.Channel {
+
+	ctxb := context.Background()
+	ctxt, cancel := context.WithTimeout(ctxb, defaultTimeout)
+	defer cancel()
+
+	channelResp, err := node.ListChannels(ctxt, &lnrpc.ListChannelsRequest{
+		ActiveOnly: true,
+	})
+	require.NoError(t, err)
+
+	chanFundingHash, err := lnrpc.GetChanPointFundingTxid(chanPoint)
+	require.NoError(t, err)
+
+	chanPointStr := fmt.Sprintf("%v:%v", chanFundingHash,
+		chanPoint.OutputIndex)
+
+	var targetChan *lnrpc.Channel
+	for _, channel := range channelResp.Channels {
+		if channel.ChannelPoint == chanPointStr {
+			targetChan = channel
+
+			break
+		}
+	}
+	require.NotNil(t, targetChan)
+
+	return targetChan
+}
+
+func assertChannelSatBalance(t *testing.T, node *HarnessNode,
+	chanPoint *lnrpc.ChannelPoint, local, remote int64) {
+
+	targetChan := fetchChannel(t, node, chanPoint)
+
+	require.InDelta(t, local, targetChan.LocalBalance, 1)
+	require.InDelta(t, remote, targetChan.RemoteBalance, 1)
+}
+
+func assertChannelAssetBalance(t *testing.T, node *HarnessNode,
+	chanPoint *lnrpc.ChannelPoint, local, remote uint64) {
+
+	targetChan := fetchChannel(t, node, chanPoint)
+
+	var assetBalance rfqmsg.JsonAssetChannel
+	err := json.Unmarshal(targetChan.CustomChannelData, &assetBalance)
+	require.NoError(t, err)
+
+	require.Len(t, assetBalance.Assets, 1)
+
+	require.InDelta(t, local, assetBalance.Assets[0].LocalBalance, 1)
+	require.InDelta(t, remote, assetBalance.Assets[0].RemoteBalance, 1)
+}
+
+func routingFee(amt lnwire.MilliSatoshi) lnwire.MilliSatoshi {
+	return amt + (amt / 1000_000) + 1000
+}
```

Comment:
consider adding a comment to explain this math: why is that the routing fee?

Reply:
Updated.